idd/Energy+.idd.in

```diff
@@ -40683,6 +40683,52 @@ AirTerminal:SingleDuct:ConstantVolume:Reheat,
       \units C
       \minimum> 0.0
 
+AirTerminal:SingleDuct:ConstantVolume:NoReheat,
+       \memo Central air system terminal unit, single duct, constant volume, without reheat coil
+  A1 , \field Name
+       \required-field
+       \reference AirTerminalUnitNames
+       \reference AFNTerminalUnitNames
+  A2 , \field Availability Schedule Name
+       \note Availability schedule name for this system. Schedule value > 0 means the system is available.
+       \note If this field is blank, the system is always available.
+       \type object-list
+       \object-list ScheduleNames
+  A3,  \field Air Outlet Node Name
+       \required-field
+       \type node
+       \note This is an air outlet node from the air distribution unit. This node name should be one of the 
+       \note supply air inlet node names of a zone served by this component.  
+  A4,  \field Air Inlet Node Name
+       \required-field
+       \type node
+       \note The air-inlet node name that connects the air splitter to the individual zone air distribution 
+       \note unit. This node should also be one of the outlet air node of an AirLoopHVAC:ZoneSplitter or 
+       \note AirLoopHVAC:SupplyPlenum component.
```

Comment:
It's always inlet then outlet. @Nigusse why are these inputs in reverse order?

Reply:
OK whew I'm not crazy :rofl:

Reply:
Um, no, about half of the vav terminal units have the outlet node first
```
AirTerminal:SingleDuct:ConstantVolume:Reheat,
  A3,  \field Air Outlet Node Name
  A4,  \field Air Inlet Node Name

AirTerminal:SingleDuct:VAV:NoReheat,
  A3,  \field Air Outlet Node Name
  A4,  \field Air Inlet Node Name

AirTerminal:SingleDuct:VAV:Reheat,
  A3 , \field Damper Air Outlet Node Name
  A4 , \field Air Inlet Node Name

AirTerminal:SingleDuct:VAV:HeatAndCool:NoReheat,
  A3,  \field Air Outlet Node Name
  A4,  \field Air Inlet Node Name

AirTerminal:SingleDuct:VAV:HeatAndCool:Reheat,
  A3,  \field Damper Air Outlet Node Name
  A4,  \field Air Inlet Node Name
AirTerminal:DualDuct:ConstantVolume,
AirTerminal:DualDuct:VAV,
AirTerminal:DualDuct:VAV:OutdoorAir,

But there are the opposite (inlet first)
AirTerminal:SingleDuct:SeriesPIU:Reheat,
AirTerminal:SingleDuct:VAV:Reheat:VariableSpeedFan,
AirTerminal:SingleDuct:ParallelPIU:Reheat,
AirTerminal:SingleDuct:ConstantVolume:FourPipeInduction,
AirTerminal:SingleDuct:ConstantVolume:FourPipeBeam,
AirTerminal:SingleDuct:ConstantVolume:CooledBeam,
```
So, it could go either way. I don't have an opinion since the existing ones are so mixed.

Reply:
This makes me uncomfortable.  I need an adult!